packages/palette/src/elements/Pagination/LargePagination.tsx

```diff
@@ -0,0 +1,155 @@
+import React from "react"
+import styled, { css } from "styled-components"
+import { color } from "../../helpers/color"
+import { ChevronIcon } from "../../svgs/ChevronIcon"
+import { Box } from "../Box"
+import { Flex } from "../Flex"
+import { Sans } from "../Typography"
+
+export interface Props {
+  onClick?: (cursor: string, page: number) => void
+  onNext?: () => void
+  pageCursors: any
+  hasNextPage: boolean
+  scrollTo?: string
+}
+
+/** LargePagination */
+export const LargePagination = (props: Props) => {
+  const {
+    pageCursors: { around, first, last, previous },
+    onClick,
+    onNext,
+    hasNextPage,
+  } = props
+
+  const handlePrevClick = () => {
+    if (previous) {
+      onClick(previous.cursor, previous.page)
+    }
+  }
+
+  const handleNextClick = () => {
+    onNext()
+  }
+
+  const aroundPages = around.map(pageCursor => {
+    const { cursor, page } = pageCursor
+    const key = cursor + page
+    return <Page key={key} onClick={onClick} pageCursor={pageCursor} />
+  })
+
+  return (
+    <Flex
+      alignItems="baseline"
+      flexDirection="row"
+      justifyContent="flex-end"
+      mr={-1}
+    >
+      {first && <FirstPage onClick={onClick} pageCursor={first} />}
+      {aroundPages}
+      {last && <LastPage onClick={onClick} pageCursor={last} />}
+
+      <Box ml={4}>
+        <PrevButton enabled={previous} onClick={handlePrevClick} />
+        <NextButton enabled={hasNextPage} onClick={handleNextClick} />
+      </Box>
+    </Flex>
+  )
+}
+
+const Page = ({ onClick, pageCursor }) => {
+  const { cursor, isCurrent, page } = pageCursor
+
+  const handleClick = () => {
+    onClick(cursor, page)
+  }
+
+  return (
+    <Button active={isCurrent} onClick={handleClick}>
+      <Sans size="3" weight="medium" display="inline">
```

Comment:
It's probably worth updating these to use `Text` component (`<Text variant="mediumText" ...`) which correspond to the updated typographic styles.

Reply:
> make no visual changes

Yep, makes sense!

Reply:
I'll come back for this in a follow up 👍